DigitalRain/GameUtilities/IGameObject.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DigitalRain.GameUtilities
+{
+    interface IGameObject
```

Comment:
I looked for this interface in the monogame library itself and was surprised not to find it.  There is something called a GameObject but it's a full class.  🤷‍♂️

Reply:
Yeah I wasn't sure what to call this. `IUpdateDraw`? Just wanted a way to easily define which classes have the update and draw methods without having to scroll all the way down.